src/v2/Components/Ad/Ad.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, memo } from "react"
```

Comment:
Is it premature to add this to `v2/Components`?

Reply:
Yeah; I suppose there's no intention to show ads anywhere else.